### ISSUE TITLE
Default value `null` does not match type `selector` in component `godrays` 

### ISSUE THREAD

**Issue by studiouniverse** [wizgrav/aframe-effects#2]
`core:schema:warn Default value 'null' does not match type 'selector' in component 'godrays' `

Get this warning on scene load, even when I seemingly am not using the godray attribute, which I'm assuming you don't _have_ to use?

(Aframe version 0.5.0)

**Comment by wizgrav** [comment]
Yes this is ok, won't affect operation. The godrays component needs an entity to act as the center of the radial blur. By default I set it to null and the schema parser whines when the component is registered. I'll see if that can be silenced but doesn't affect operation in any case and you don't have to use godrays.

On a unrelated note how do you use effects with 0.5.0, it's missing the functionality required. Maybe you mean current aframe master?
